students/fortucj/lesson02/Lesson02_exercise1_Grid_Printer.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+print("""Grid Printer exercise
+
+part1()
+print_grid(arg1)
+print_grid2(arg2,arg3))
+
+All entries must be real integers.
+Non-real integer entries will return a correction message.
+
+arg1 must also be an odd number.
+Even numbers will return a correction message.
+
+Negative numbers are ok.""")
+
+import math
+
+def part1():
+    cells=2
```

Comment:
You should have spaces around the assignment operator, here and all similar below